framework/Dockerfile

```diff
@@ -32,12 +32,10 @@ LABEL org.opencontainers.image.description="An image prepared for api-bench usag
 ENV IDRINTH_API_BENCH_VERSION=$BUILD_VERSION
 ENV IDRINTH_API_BENCH_BUILT=$BUILD_TIME
 
-RUN mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench
-RUN git clone https://github.com/idrinth-api-bench/framework.git \
+RUN mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench/
+RUN git clone https://github.com/idrinth-api-bench/framework.git /project\
     && npm ci \
```

Comment:
I assume a cd is missing in between? Otherwise you are running the following commands NOT where you cloned things

Reply:
added cd